Sming/SmingCore/Network/TcpClient.h

```diff
@@ -54,6 +54,13 @@ class TcpClient : public TcpConnection
 	__forceinline bool isProcessing()  { return state == eTCS_Connected || state == eTCS_Connecting; }
 	__forceinline TcpClientState getConnectionState() { return state; }
 
+	inline void setReceiveCallback( TcpClientDataDelegate on_receive ) {
```

Comment:
Please, replace `inline` with `__forceinline` to be consistent with the other inline methods.